mix.exs

```diff
@@ -9,6 +9,8 @@ defmodule Lob.Mixfile do
       preferred_cli_env: ["coveralls.html": :test],
       start_permanent: Mix.env == :prod,
       test_coverage: [tool: ExCoveralls],
+      description: "Lob Elixir Library.",
```

Comment:
Is the period necessary?

Reply:
Mark mentioned it in a previous PR. Up to you guys. I'll remove for now.